libbeat/logp/file_rotator.go

```diff
@@ -67,7 +67,6 @@ func (rotator *FileRotator) WriteLine(line []byte) error {
 		}
 	}
 
-	line = append(line, '\n')
```

Comment:
It is not 100% clear to me what this change was needed. Seems like before there was no newline at the end of each event and now there is.